File: auth/src/main/java/eu/tsp/smartappartment/auth/entity/AuthResponse.java
package eu.tsp.smartappartment.auth.entity;


import lombok.Data;

@Data
public class AuthResponse
{
    private String role;
    private int roomId;
    private int personId;
    private String room;
    private String livingRoom;
    private String kitchen;
    private String name;
}

File: flutter-iot-app-ui-main/lib/models/auth_response.dart
import 'package:json_annotation/json_annotation.dart';

part 'auth_response.g.dart';

@JsonSerializable()
class Auth_response {
  Auth_response();

  late String role;
  late num roomId;
  late num personId;
  late String room;
  late String livingRoom;
  late String kitchen;
  late String name;
  
  factory Auth_response.fromJson(Map<String,dynamic> json) => _$Auth_responseFromJson(json);
  Map<String, dynamic> toJson() => _$Auth_responseToJson(this);
}
